src/test/java/Calculator/TestCalculator.java

```diff
@@ -0,0 +1,78 @@
+package Calculator;
+
+import Calculator.View.InputView;
+import Calculator.Model.Calculator;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestCalculator {
+    Calculator calculator;
+
+    @BeforeEach
+    public void setUp() {
+        calculator = new Calculator();
+    }
+
+    @Test
+    @DisplayName("callCalculator 덧셈 정상 확인")
+    public void callCalculator_add() {
+        String expected = "1";
+        String actual = calculator.callCalculator("0", "+", "1");
+        assertThat(actual).isEqualTo(expected);
+    }
+
+    @Test
+    @DisplayName("callCalculator 뺄셈 정상 확인")
+    public void callCalculator_subtract() {
+        String expected = "1";
+        String actual = calculator.callCalculator("2", "-", "1");
+        assertThat(actual).isEqualTo(expected);
+    }
+
+    @Test
+    @DisplayName("callCalculator 곱셈 정상 확인")
+    public void callCalculator_multiple() {
+        String expected = "6";
+        String actual = calculator.callCalculator("3", "*", "2");
+        assertThat(actual).isEqualTo(expected);
+    }
+
+    @Test
+    @DisplayName("callCalculator 나눗셈 정상 확인")
+    public void callCalculator_divide() {
+        String expected = "2";
+        String actual = calculator.callCalculator("10", "/", "5");
+        assertThat(actual).isEqualTo(expected);
+    }
+
+    @Test
+    @DisplayName("calculator 숫자 입력")
+    public void calculator_number() {
+        String expected = "2";
+
+        ArrayList<String> input = new ArrayList<>();
+        input.add("10");
+        input.add("/");
+        input.add("5");
+        ArrayList<String> actual = calculator.calculate(input);
+        assertThat(calculator.getCurrentResult(actual)).isEqualTo(expected);
```

Comment:
```suggestion
        List<String> input = CalculateFixture.of("10", "/", "5");
        List<String> actual = calculator.calculate(input);
        assertThat(calculator.getCurrentResult(actual)).isEqualTo(expected);
```
- 단위 테스트 내에서 값을 바인딩하기 보다는 별도로 메소드를 분리하거나 Fixture를 구성하는건 어떨까요